doc/tutorial/checker.py

```diff
@@ -1,3 +1,10 @@
+from pathlib import Path
+
 def task_checker():
     return {'actions': ["pyflakes sample.py"],
             'file_dep': ["sample.py"]}
+
+def task_checker_pathlib():
```

Comment:
I agree - it is a good thing to concentrate reader on one feature at once. I can remove this example here and add a new section like `pathlib support` with example which uses `pathlib` both in `file_dep`, `targets` and `CmdAction`.